### PR TITLE
Refine org view into tabs

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
The first step to enriching the organizations

### Changes proposed in this pull request
Move courses and request access into tabs reach for enrichment feature

### Guidance to review
# After
screen shot 2018-07-27 at 10 11 14
screen shot 2018-07-27 at 10 11 02
screen shot 2018-07-27 at 10 11 35

### CODE REVIEW

**Comment by fofr** [comment]
Why is "Organisation" capitalised in the URLs in your screenshot?

**Comment by dankmitchell** [comment]
@defong can we make all URL's lowercase and hyphenated, please?

**Comment by defong** [comment]
Done lowercase

File: src/ui/Controllers/OrganisationController.cs [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            };
+
+            return result;
+        }
+
+        [HttpGet]
+        [Route("{ucasCode}/request-access")]
+        public async Task<ViewResult> RequestAccess(string ucasCode)
+        {
+            var tabViewModel = await GetTabViewModelAsync(ucasCode, "request-access");
```
[the remark this answers is not in the document]
Comment (defong) [reply src/ui/Controllers/OrganisationController.cs]: Fragile but tested.

File: src/ui/ViewModels/TabbedViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ManageCourses.ApiClient;
+
+namespace GovUk.Education.ManageCourses.Ui.ViewModels
+{
+    public class TabbedViewModel
```
[the remark this answers is not in the document]
Comment (defong) [reply src/ui/ViewModels/TabbedViewModel.cs]: Its used here
`_TabViewLayout.cshtml`

It's inheritance in order to allow for composition, hence composition over inheritance.

The layout uses inheritance.

Child.cshtml `this is where the parent view model`
_TabViewLayout.cshtml 'this renders the section'
_OrganisationDetailsLayout.cshtml 'this declares the section'
_Layout.cshtml
_Layout_base.cshtml

File: src/ui/Controllers/OrganisationController.cs [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    {
+        private readonly IManageApi _manageApi;
+
+        public OrganisationController(IManageApi manageApi)
+        {
+            _manageApi = manageApi;
+        }
+
+        [Route("{ucasCode}/courses")]        
+        public async Task<IActionResult> Courses(string ucasCode)
```
[the remark this answers is not in the document]
Comment (defong) [reply src/ui/Controllers/OrganisationController.cs]: See below

**Pull request merged**